.travis.yml

```diff
@@ -4,17 +4,34 @@ sudo: false
 cache:
   - pip
 
+before_cache:
+  - rm ~/.cache/pip/log/debug.log
+
 language: python
 python:
   - "2.7"
   - "3.5"
 
 install:
-  - pip install -r requirements.txt
   - pip install tox-travis
 
 script:
-  # TODO(rhcarvalho): check syntax of other important entrypoint playbooks
-  - ansible-playbook --syntax-check playbooks/byo/config.yml
   - tox
-  - cd utils && tox
+
+notifications:
+  email:
+    recipients:
+      - jdetiber@redhat.com
+      - sdodson@redhat.com
```

Comment:
@ashcrow so, these notifications are for when the branch is broken post-commit. Mainly needed because of the issues with generated modules being out of sync in master due to conflicting PRs getting merged (where they individually passed generate_validation testing).